_events/2017-02-13-test5

```diff
@@ -0,0 +1,7 @@
+---
+date: 13/02/2017
```

Comment:
Use a different property then date (e.g., `eventdate`) and use ISO8601 for the date format (2017-02-13)